### PR TITLE
Bug 1910665: Kuryr: Add note about expected OpenStack subnet

### PR DESCRIPTION
**Pull request by dulek**
With Kuryr the subnet created for the services will be twice the size of
the subnet specified in install-config.yaml which might be
counterintuitive for the users. This commit adds a note about it.

### CODE REVIEW

**Comment by dulek** [comment]
@maxwelldb: Hm, I'm not sure how it's supposed to work with bugs on docs. Can you advise?

**Comment by maxwelldb** [comment]
@dulek Can't say I know why the bot hates this, but NBD. It's fine. 

Who should the QE contact for this change be?

**Comment by dulek** [comment]
> @dulek Oh, and this is 4.5+ rather than just 4.5, right?

Yup!

> @dulek Can't say I know why the bot hates this, but NBD. It's fine.
> 
> Who should the QE contact for this change be?

I guess we can ask @itzikb-redhat for a favor? It's very simple to verify if what I wrote is correct - just compare the CIDR of service subnet in openstack with the `serviceNetwork` setting on any environment.

File: modules/installation-osp-kuryr-config-yaml.adoc
Comment (maxwelldb) [inline modules/installation-osp-kuryr-config-yaml.adoc:57]: Does something like this work for you?

> The Amphora Octavia driver creates two ports per load balancer. As a result, the service subnet that the installer creates is twice the size of the CIDR that is specified as the `serviceNetwork` property. The larger range is required to prevent IP address conflicts.

**Review by maxwelldb — CHANGES_REQUESTED** [review]

File: modules/installation-osp-kuryr-config-yaml.adoc
Comment (dulek) [reply modules/installation-osp-kuryr-config-yaml.adoc]: Sure, I just updated the patch with the suggestion.

**Comment by itzikb-redhat** [comment]
LGTM @maxwelldb Is there a way to see it as a part of a doc?

**Comment by maxwelldb** [comment]
@itzikb-redhat https://elegant-dubinsky-38dc0b.netlify.app/upi-kuryr.html#installation-osp-kuryr-config-yaml_installing-openstack-user-kuryr

**Comment by itzikb-redhat** [comment]
LGTM.Thanks

**Comment by maxwelldb** [comment]
Thanks!

File: modules/installation-osp-kuryr-config-yaml.adoc
Comment (maxwelldb) [inline modules/installation-osp-kuryr-config-yaml.adoc:59]:
```suggestion
CIDR that is specified as the value of the `serviceNetwork` property. The larger range is
```

**Review by maxwelldb — APPROVED** [review]

**Comment by maxwelldb** [comment]
@dulek Might want to push a trivial change to re-trigger CI. [Last build](https://travis-ci.com/github/openshift/openshift-docs/builds/212847987) failed through no fault of your own.

**Comment by maxwelldb** [comment]
https://github.com/openshift/openshift-docs/pull/28358#issuecomment-758739084

@dulek Making a trivial change or just rebasing your two commits into one should trigger a new build. The latter's required to merge this, anyway.

**Comment by maxwelldb** [comment]
@dulek ^^^

**Comment by dulek** [comment]
Does such rebase work?

**Comment by maxwelldb** [comment]
@dulek Somehow missed this. Build successful. :+1:

**Comment by maxwelldb** [comment]
@openshift/team-documentation PTAL. Thanks!

**Comment by maxwelldb** [comment]
@openshift/team-documentation PTAL at this PR. Thanks!

File: modules/installation-osp-kuryr-config-yaml.adoc
Comment (kalexand-rh) [reply modules/installation-osp-kuryr-config-yaml.adoc:58]: I agree with this suggestion.

**Review by kalexand-rh — COMMENTED** [review]
I think that the suggestion that you made needs to be incorporated, but it otherwise LGTM.

**Comment by maxwelldb** [comment]
@kalexand-rh Thanks!

**Comment by maxwelldb** [comment]
@dulek I'm just now noticing that this targets 4.7. 

The bug is for 4.5, right? 

If just 4.5, we need to retarget at the `enterprise-4.5` branch.

If it's 4.5+, we need to retarget it at `master`. After that's merged, there's a cherrypick process to follow to get the change onto 4.5, 4.6, 4.7, and the WIP 4.8 branches. 

Either way, there might be some conflicts to resolve. :)

**Comment by dulek** [comment]
This is 4.5+, let me attempt retargetting this then.

**Comment by dulek** [comment]
Alright, this now targets master.

**Comment by maxwelldb** [comment]
@dulek Thanks!

**Comment by maxwelldb** [comment]
@dulek All done. The change should be reflected in the docs shortly.

**Pull request merged**
